teachertool/src/constants.ts

```diff
@@ -17,8 +17,8 @@ export namespace Strings {
     export const ChecklistNamePlaceholder = lf("Enter Checklist Name");
     export const AddCriteria = lf("Add Criteria");
     export const Actions = lf("Actions");
-    export const AutoRun = lf("Auto-run");
-    export const AutoRunDescription = lf("Automatically re-evaluate when the checklist or project changes");
+    export const RunOnLoad = lf("Eval on load");
```

Comment:
Will changing this to "Evaluate on Load" be too long? It's a bit of a nitpick, but I almost regard "eval" as short for "evaluation" rather than "evaluate", so it feels a bit off to be used here.

Reply:
Sure, I can change it to "Evaluate". It seems clearer. Later when we address responsiveness we might want to dynamically shorten it when the panel gets narrow.

Reply:
+1 for Evaluate (or "Run Checks on Load", but I think Evaluate on Load is more concise)